src/test/librados_test_stub/TestWatchNotify.cc

```diff
@@ -0,0 +1,221 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "test/librados_test_stub/TestWatchNotify.h"
+#include "include/Context.h"
+#include "common/Finisher.h"
+#include <boost/bind.hpp>
+#include <boost/function.hpp>
+
+namespace librados {
+
+class FunctionContext : public Context {
```

Comment:
maybe add this to Context.h - it's pretty handy, and it was just added in ImageWatcher